src/components/FeatureViews/Accounts/TabComponents/EnrollmentView.js

```diff
@@ -382,22 +381,24 @@ const CourseSessionStatus = () => {
                             Enrollment Balance Left: ${enrollment.balance}
                         </Typography>
                     </Grid>
-                    <Grid item xs={3}>
-                        <FormControl component="fieldset">
-                            <FormGroup>
-                                <FormControlLabel
-                                    control={
-                                        <Switch
-                                            checked={highlightSession}
-                                            onChange={handleHighlightSwitch}
-                                            color="primary"
-                                            value="upcoming-session"
+                    <Grid item xs={3} container alignItems="flex-start">
+                            <Grid item>
```

Comment:
This (and its contents) is indented one level too many, dedent for better alignment